src/app/frank/fd_frank_quic.c

```diff
@@ -133,13 +133,14 @@ static long allow_syscalls[] = {
   __NR_nanosleep, /* fd_tempo_tick_per_ns calibration */
   __NR_getpid,    /* OpenSSL RAND_bytes checks pid, temporarily used as part of quic_init to generate a certificate */
   __NR_getrandom, /* OpenSSL RAND_bytes reads getrandom, temporarily used as part of quic_init to generate a certificate */
+  __NR_sendto,    /* fd_xsk requires sendto */
 };
 
 fd_frank_task_t frank_quic = {
   .name     = "quic",
   .in_wksp  = NULL,
   .out_wksp = "quic_verify",
-  .close_fd_start = 4, /* stdin, stdout, stderr, logfile */
+  .close_fd_start = 6, /* stdin, stdout, stderr, logfile, network namespace, xsk->xsk_fd */
```

Comment:
Is the network namespace fd still open here? Or the kernel just doesn't reuse the fd?

Reply:
Updated to close the network namespace FD.